Snowflake.Data.Tests/Snowflake.Data.Tests.csproj

```diff
@@ -31,8 +31,9 @@
   </ItemGroup>
   
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.2'">
+    <PackageReference Include="Microsoft.NetCore.App" Version="2.2.8" />
     <PackageReference Include="Microsoft.PowerShell.Commands.Utility">
-      <Version>6.1.0-rc.1</Version>
+      <Version>6.2.3</Version>

```

Comment:
_Microsoft powershell needs to be updated from 6.1.0-rc.1 to either 6.1.6 or 6.2.3_

Updating to 6.2.3